pipelines/deployer/deployer.yaml

```diff
@@ -778,6 +778,23 @@ jobs:
     config: *generate_users_terraform
     params:
       ACCOUNT_NAME: ((account-name))
+  - task: remove-prevent-destroy
+    image: task-toolbox
+    timeout: 10s
+    config:
+      platform: linux
+      run:
+        path: /bin/bash
+        args:
+        - -euo
+        - pipefail
+        - -c
+        - |
+          find platform -name '*.tf' | xargs sed -i'' -e '/prevent_destroy = true/d'
```

Comment:
Do we want to be more surgical here? Ie do we want to target a particular file to edit rather than just blanketly removing `prevent_destroy = true` across the whole codebase?